### PR TITLE
Full stack Capstone Project: Jobforcash

### PR DESCRIPTION
**Pull request by V0pe**
In this project, we achieved building the website using the following milestone:
The [frontend application](https://github.com/V0pe/jobforcash_frontend) is here.
- [ ] The user logs in to the website, only by typing the username and password.
- [ ] In the navigation panel, the user can see links to:
    - [ ] Laborers seen as items
    - [ ] Reserve" form.
    - [ ] My reservations".
    - [ ] Add motorcycle/doctor/class/item that you selected as a theme" (the link is visible to everybody).
    - [ ] Delete motorcycle/doctor/class/item that you selected as a theme" (the link is visible to everybody).
- [ ] On the main page, the user can see a laborers (https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005577).
- [ ] When the user selects a specific item, they can see the [details page](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005579) with its full description (skip the "Rotate image" button).
In the details page, the user can click the "Reserve" button (in the design you can see the "Configure" button - please replace it with the "Reserve" button).
- [ ] Details
   - [ ] When the user clicks the "Add item" link in the navigation panel they can see a form for adding a new item.
- [ ] Make the app responsive, creating both mobile and desktop versions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,206 @@
...
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd jobforcash_backend
+  bundle install
+```
```
Comment (Meltrust) [inline README.md:121]: [ ] Kindly, include in your instructions the required db commands that your users will need to run your backend app: `rails db:create, rails db:migrate, rails db:seed`. 👍

File: README.md [hunk 2]
```diff
@@ -1,24 +1,206 @@
...
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+  rspec
+```
```
Comment (Meltrust) [inline README.md:138]: - [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @V0pe,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:

-  Tests are passing✔️
-  Nice code organization ✔️ 




#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

File: README.md (hunk 2 above)
Comment (German-Cobian) [reply README.md:138]: The test folder has been delete.

File: README.md (hunk 1 above)
Comment (German-Cobian) [reply README.md:121]: All the commands that users will need to run the backend app are now included.

**Review by Meltrust — APPROVED** [review]
#### Hi @V0pe ,


# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause

### To highlight

- Readme has been fixed ✔️ 
- Superfluous test folder is being removed ✔️ 



Cheers and Happy coding!👏👏👏


------

**Pull request merged**